config/hackathon.yml

```diff
@@ -2,15 +2,20 @@ defaults: &defaults
   # Logic config
   registration_is_open: false
   event_is_over: true
-  last_day_to_apply: <%= Date.new(2017, 2, 11) %>
+  last_day_to_apply: <%= Date.new(2018, 1, 28) %>
```

Comment:
The date shouldn't be January 28, it should be earlier

Reply:
What date should it be instead?

I chose the 28th more as a temporary date (no one would be applying after the day-of) unless there's a hard cut-off we want.

Reply:
Follow-up Slack conversation: decided on Jan 22nd